Sources/Fuzzilli/FuzzIL/Operations.swift

```diff
@@ -279,6 +279,24 @@ class CreateTemplateString: Operation {
     }
 }
 
+class CallTaggedTemplate: Operation {
+    // Stores the string elements of the temaplate literal
+    let parts: [String]
+
+    var numInterpolatedValues: Int {
+        return numInputs - 1
+    }
+
+    // This operation isn't parametric since it will most likely mutate imported templates (which would mostly be valid JS snippets) and
```

Comment:
I'm wondering if this is really necessary (also for `CreateTemplateString`). _If_ a string really is valid JavaScript and is subsequently executed, and we mutate it, we'll just end up with an invalid program and undo the mutation. On the other hand, I feel like if we mutate regular strings, we should also mutate template strings. WDYT?
An alternative would be to extend the existing `CodeString` mechanism to support interpolation, but in that case the FuzzIL compiler would have to know when a string is supposed to be JavaScript code, which probably isn't really feasible.

Reply:
I think the reason we didn't want to do this was due to the high failure rate it would have. I'm happy to change that to enable mutations.
> An alternative would be to extend the existing CodeString mechanism to support interpolation, but in that case the FuzzIL compiler would have to know when a string is supposed to be JavaScript code, which probably isn't really feasible.

Yeah that's going to require some gnarly hacks to parse and compile :(

Reply:
Unfortunately no, I don't have data on this.

Reply:
@saelo any thoughts on the above? I can start gathering stats if we need that to make changes to this PR

Reply:
Yeah I'd suggest something like running 4-8 Instances of Fuzzilli, half of them with mutation of TaggedTemplateStrings allowed, the other half with it forbidden, and keeping track of the failure rates. If they are not significantly different, I'd prefer if TaggedTemplates were mutable as well (I guess that includes the existing TemplateStrings: https://github.com/googleprojectzero/fuzzilli/blob/4b0df78015bbfb41ff261d366683e3d6f39b3eee/Sources/Fuzzilli/FuzzIL/Operations.swift#L273)

Reply:
Sounds good to me, any ideas on capturing failure rates? I could add a counter to record valid and invalid program generation from the two generators (tagged template and template string). I could also just use the existing correctness rate but that's probably less accurate.

Reply:
With some of the recent changes, if you add --logLevel=verbose, you'll get detailed statistics about the correctness rates of every CodeGenerator, Mutator, etc. That's probably good enough